.github/workflows/auto-author-assign.yml

```diff
@@ -0,0 +1,14 @@
+name: Auto Author Assign
+
+on:
+  pull_request_target:
+    types: [opened, reopened]
```

Comment:
I'm not sure we want the automation on reopen but let's keep it for now and see how it goes. An author should already be assigned in that case anyway.

Reply:
Yes, you are right about it. The good thing about this action is if the PR is already assigned or the PR is created by the bot (like _dependabot_), then it skips the auto-assignment process. :)